applications/api-cat/src/test/java/no/acat/harvester/ApiHarvestTest.java

```diff
@@ -21,20 +22,20 @@
     @Test
     public void harvestAllOK() throws Throwable {
 
-        ElasticsearchService elasticsearchService = mock(ElasticsearchService.class);
-        ApiHarvester harvester = new ApiHarvester(elasticsearchService);
-
-        ApiHarvester spyHarvester = spy(harvester);
-        doNothing().when(spyHarvester).indexApi(any());
-
-        ApiDocumentBuilder mockApiDocumentBuilder = mock(ApiDocumentBuilder.class);
-        doReturn(new ApiDocument()).when(mockApiDocumentBuilder).create(any());
-
-        doReturn(mockApiDocumentBuilder).when(spyHarvester).createApiDocumentBuilder();
-
-        List<ApiDocument> response = spyHarvester.harvestAll();
-        final int FROM_CSV = 11;
-        assertThat(response.size(), is(FROM_CSV));
+//        ElasticsearchService elasticsearchService = mock(ElasticsearchService.class);
```

Comment:
Vi bør aldri kommentere ut kode som dette, denne bør slettes. Om du glemmer å fjerne denne kan neste utvkler ikke vite om denne koden er HELT ok, og bare midlertidig "deaktivert" eller er dårlig og må aldri kjøres. Om du er redd for å miste koden, så frykt ikke, den er jo i forrige commit i github :)